src/snowflake/connector/connection.py

```diff
@@ -256,6 +256,18 @@ def _get_private_bytes_from_file(
         True,
         bool,
     ),  # Enable sending retryReason in response header for query-requests
+    "session_token": (
+        None,
+        (type(None), str),
+    ),  # session token from another connection, to be provided together with master token
+    "master_token": (
+        None,
+        (type(None), str),
+    ),  # master token from another connection, to be provided together with session token
+    "master_validity_in_seconds": (
```

Comment:
does the session token has a similar validity in seconds concept?

Reply:
I don't think it does--or if it does, it's not something the connector code ever sees. This value I get from the node connector when I serialize the object to get the session, master, and master validity. It has no other info on it. 

Example from the python connector code, it references master validity but nothing about session. 
https://sourcegraph.c2.us-central1.gcp-dev.app.snowflake.com/github.com/snowflakedb/snowflake-connector-python/-/blob/src/snowflake/connector/network.py?L408:16